plutus-benchmark/nofib/test/9.6/clausify-F5.pir.golden

```diff
@@ -1,990 +1,1001 @@
-                    defaultBody)
-                 (\(default_arg0 : Formula) (default_arg1 : Formula) ->
-                    defaultBody)
-                 (\(default_arg0 : Formula) -> defaultBody)
-                 (\(s : integer) ->
-                    Tuple2_match
-                      {List integer}
-                      {List integer}
-                      x
-                      {Tuple2 (List integer) (List integer)}
-                      (\(c : List integer) (a : List integer) ->
-                         Tuple2
-                           {List integer}
-                           {List integer}
-                           c
-                           (insert {integer} `$fOrdInteger` s a))))
-            (\(s : integer) ->
-               Tuple2_match
-                 {List integer}
-                 {List integer}
-                 x
-                 {Tuple2 (List integer) (List integer)}
-                 (\(c : List integer) (a : List integer) ->
-                    Tuple2
-                      {List integer}
-                      {List integer}
-                      (insert {integer} `$fOrdInteger` s c)
-                      a))
-  in
-  let
-    !`$p1Ord` : all a. Ord a -> (\a -> a -> a -> Bool) a
-      = /\a ->
-          \(v : Ord a) ->
-            Ord_match
-              {a}
-              v
-              {(\a -> a -> a -> Bool) a}
-              (\(v : (\a -> a -> a -> Bool) a)
-                (v : a -> a -> Ordering)
-                (v : a -> a -> Bool)
-                (v : a -> a -> Bool)
-                (v : a -> a -> Bool)
-                (v : a -> a -> Bool)
-                (v : a -> a -> a)
-                (v : a -> a -> a) ->
-                 v)
-    !compare : all a. Ord a -> a -> a -> Ordering
-      = /\a ->
-          \(v : Ord a) ->
-            Ord_match
-              {a}
-              v
-              {a -> a -> Ordering}
-              (\(v : (\a -> a -> a -> Bool) a)
-                (v : a -> a -> Ordering)
-                (v : a -> a -> Bool)
-                (v : a -> a -> Bool)
-                (v : a -> a -> Bool)
-                (v : a -> a -> Bool)
-                (v : a -> a -> a)
-                (v : a -> a -> a) ->
-                 v)
-  in
-  letrec
-    !`$fOrdList_$ccompare` : all a. Ord a -> List a -> List a -> Ordering
-      = /\a ->
-          \(`$dOrd` : Ord a) (ds : List a) (ds : List a) ->
-            List_match
-              {a}
-              ds
-              {all dead. Ordering}
-              (/\dead ->
-                 List_match
-                   {a}
-                   ds
-                   {all dead. Ordering}
-                   (/\dead -> EQ)
-                   (\(ds : a) (ds : List a) -> /\dead -> LT)
-                   {all dead. dead})
-              (\(ds : a) (ds : List a) ->
-                 /\dead ->
-                   List_match
-                     {a}
-                     ds
+((let
```

Comment:
Looks like this file now has an additional pair of parentheses (it's hard to tell whether that's the only difference, but seems so?), which is redundant